packages/contracts-bedrock/src/universal/CrossDomainMessenger.sol

```diff
@@ -115,6 +116,18 @@ abstract contract CrossDomainMessenger is
     ///         call in `relayMessage`.
     uint64 public constant RELAY_GAS_CHECK_BUFFER = 5_000;
 
+    /// @notice Base gas required for any transaction in the EVM.
+    uint64 public constant TX_BASE_GAS = 21_000;
+
+    /// @notice Floor overhead per byte of non-zero calldata in a message. Calldata floor was
+    ///         introduced in EIP-7623.
+    uint64 public constant FLOOR_CALLDATA_OVERHEAD = 40;
+
+    /// @notice Overhead bytes regardless of the size of the message. Value is specific to the V1
+    ///         message encoding scheme. 260 is an upper bound, actual overhead can be as low as
+    ///         228 bytes for an empty message.
+    uint64 public constant ENCODING_OVERHEAD = 260;
```

Comment:
From this comment I don't fully follow where the 260 comes from or what this value is for

Reply:
Will clarify. 260 is basically all of the bytes when you do `abi.encodeCall(relayMessage, ...)` that are NOT the actual _message itself.

Reply:
The point of this is that you really need to be computing the calldata overhead based on the fully encoded message (`abi.encodeCall(relayMessage, ....)`) but this interface is already used all over the place so we only get the internal `_message` data and not the abi encoded thing. As a result, we want to figure out how much overhead ABI encoding adds to the `_message`.

Reply:
Updated the message to clarify this a bit